composer.json

```diff
@@ -10,6 +11,16 @@
         "jalno/lumen": "^0.0.1",
         "webonyx/graphql-php": "^14.8.0"
     },
+    "require-dev": {
+        "nunomaduro/larastan": "^0.7.6",
```

Comment:
Why did you need larastan?

Reply:
I need `larastan` for this line:
https://github.com/Jalno/graphql/blob/0d7111a7d57708d0db78ebfe4c1789e254116208/src/Http/Controllers/GraphqlController.php#L106

The `phpstan` is not understood `response()` function.